sdk/storage/Microsoft.Azure.WebJobs.Host.EndToEndTests/tests/StorageAccountTests.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+using Azure.Storage.Queues;
+using System.Linq;
+using Azure.Storage.Queues.Models;
+using Azure.WebJobs.Extensions.Storage.Common.Tests;
+using Microsoft.Azure.WebJobs.Extensions.Storage.Common;
+
+namespace Microsoft.Azure.WebJobs.Storage.IntegrationTests
+{
+    [Trait("SecretsRequired", "true")]
```

Comment:
Might need to eventually onboard these tests onto out live test infra.

Reply:
I got rid of this attribute since it doesn't really do anything. For the live tests, yes we do have a plan to glue it with our test infra, but that's after NUnit migration.